psi4/src/psi4/lib3index/cholesky.h

```diff
@@ -29,6 +29,7 @@
 #ifndef THREE_INDEX_CHOLESKY
 #define THREE_INDEX_CHOLESKY
 
+#include "psi4/psi4-dec.h"
```

Comment:
I think `psi4/pragma.h` is a far smaller header if you're just pulling in the `PSI_API` macro and will help keep compile time down.